bundles/org.openhab.binding.govee/README.md

```diff
@@ -0,0 +1,130 @@
+- H7042 LED Outdoor Bulb String Lights
+- H705A Permanent Outdoor Lights 30M
+- H705B Permanent Outdoor Lights 15M
+- H7050 Outdoor Ground Lights 11M
+- H7051 Outdoor Ground Lights 15M
+- H7055 Pathway Light
+- H7060 LED Flood Lights (2-Pack)
+- H7061 LED Flood Lights (4-Pack)
+- H7062 LED Flood Lights (6-Pack)
+- H7065 Outdoor Spot Lights
+- H6051 Aura - Smart Table Lamp
+- H6056 H6056 Flow Plus
+- H6059 RGBWW Night Light for Kids
+- H618F RGBIC LED Strip Lights
+- H618E LED Strip Lights 22m
+- H6168 TV LED Backlight
+
+## Discovery
+
+Discovery is done by scanning the devices in the Thing section. 
+
+The devices _do not_ support the LAN API support out-of-the-box.
+To be able to use the device with the LAN API, the following needs to be done (also see the "Preparations for LAN API Control" section in the [Goveee LAN API Manual](https://app-h5.govee.com/user-manual/wlan-guide)):
+
++ Start the Govee APP and add / discover the device (via Bluetooth) as described by the vendor manual
+Go to the settings page of the device 
+![govee device settings](doc/device-settings.png)
++ Note that it may take several(!) minutes until this setting comes up.
++ Switch on the LAN Control setting.
++ Now the device can be used with openHAB.
++ The easiest way is then to scan the devices via the SCAN button in the thing section of that binding
+
+## Thing Configuration
+
+Even though binding configuration is supported via a thing file it should be noted that the IP address is required and there is no easy way to find out the IP address of the device.
+One possibility is to look for the MAC address in the Govee app and then looking the IP address up via: 
+
+```shell
+arp -a | grep "MAC_ADDRESS"
+```
+
+### `govee-light` Thing Configuration
+
+| Name            | Type    | Description                           | Default | Required | Advanced |
+|-----------------|---------|---------------------------------------|---------|----------|----------|
+| hostname        | text    | Hostname or IP address of the device  | N/A     | yes      | no       |
+| macAddress      | text    | MAC address of the device             | N/A     | yes      | no       |
+| deviceType      | text    | The product number of the device      | N/A     | yes      | no       |
+| refreshInterval | integer | Interval the device is polled in sec. | 5       | no       | yes      |
+
+
+## Channels
+
+| Channel               | Type              | Read/Write | Description         |
+|-----------------------|-------------------|------------|---------------------|
+| brightness            | Percentage        | RW         |                     |
+|                       | Switch            | RW         | Power On / OFF      |
+| color                 | Color HSB Type    | RW         |                     |
+| color-temperature     | Percentage        | RW         |                     |
+| color-temperature-abs | Color Temperature | RW         | in 2000-9000 Kelvin |
```

Comment:
The Type column seems a bit confusing/mixed. What is it actually?

Reply:
The intent is to tell the user what item types are supported. I always get questions e.g. where the switch channel is. I reworked a bit.

Reply:
But they are not item types? For color temperature that would be `Dimmer`. For color it would be `Color`. And for brightness also `Dimmer`. So I don't really get it that `color-temperature` is type "Color Temperature Percentage", or that `color` is type "Color HSB Type". Perhaps description could be used for distinguishing/explaining the difference between `color-temperature` and `color-temperature-abs`?

Reply:
Finally I got all working. So the binding should be now finished.